src/ai/backend/manager/api/session.py

```diff
@@ -72,6 +72,7 @@
     SessionStartedEvent,
     SessionSuccessEvent,
     SessionTerminatedEvent,
+    KernelPullProgressEvent
```

Comment:
Though our code style checker does not check this, please add a trailing comma in multi-line lists, tuples, dicts, and function arguments always.
```suggestion
    KernelPullProgressEvent,
```